src/collective/vdexvocabulary/term.py

```diff
@@ -1,24 +1,33 @@
+# coding=utf-8
 from zope.interface import directlyProvides
-from zope.interface import implements
+from zope.interface import implementer
 from zope.schema.interfaces import ITitledTokenizedTerm
 from zope.schema.interfaces import ITokenizedTerm
 from zope.schema.vocabulary import getVocabularyRegistry
+import six
 
+try:
+    from Products.CMFPlone.utils import safe_nativestring
+except ImportError:
+    if six.PY2:
+        from Products.CMFPlone.utils import safe_encode as safe_nativestring
+    else:
+        from Products.CMFPlone.utils import safe_unicode as safe_nativestring
```

Comment:
We don't have a dependency on Products.CMFPlone. I think it's intentional that you can install the package in a pure Zope environment. If we want to drop that option, we at least need to declare the dependency on Plone in setup.py.

Reply:
I replaced `safe_nativestring` with `six.ensure_str`.